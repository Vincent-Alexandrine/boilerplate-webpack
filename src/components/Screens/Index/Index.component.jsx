import React, { PureComponent } from 'react';
import PropTypes from 'prop-types';

import Styled from './Index.styled.js';

class Index extends PureComponent {
  static defaultProps = {

  };

  static propTypes = {

  };

  render() {
    return (
      <Styled>
        <h3>ooooyyyee</h3>
      </Styled>
    );
  }
}

export default Index;
