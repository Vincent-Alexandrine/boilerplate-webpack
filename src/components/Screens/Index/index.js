import Index from './Index.component.jsx';

export default Index;
