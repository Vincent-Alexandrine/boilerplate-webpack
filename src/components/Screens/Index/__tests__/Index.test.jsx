import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import 'jest-styled-components';

import Index from '../Index.component.jsx';
import Styled from '../Index.styled.js';

test('Index component should match snapshot', () => {
  const component = renderer
    .create(
      <MemoryRouter>
        <Index />
      </MemoryRouter>
    );
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});

test('Index styled component should match snapshot', () => {
  const component = renderer
    .create(<Styled />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
