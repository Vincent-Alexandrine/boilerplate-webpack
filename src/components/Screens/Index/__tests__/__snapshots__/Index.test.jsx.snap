// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index component should match snapshot 1`] = `
<div
  className=""
>
  <main>
    <div
      className=""
    >
      <h1>
        welcome.h1
      </h1>
      <h2>
        welcome.h2
      </h2>
      <h3>
        welcome.h3
      </h3>
      <h4>
        welcome.h4
      </h4>
      <h5>
        welcome.h5
      </h5>
      <h6>
        welcome.h6
      </h6>
      <br />
      <div>
        welcome.div
        <span>
          <i>
            welcome.i
            <a
              href="http://khaledipsum.com/"
            >
              welcome.a
            </a>
          </i>
        </span>
      </div>
    </div>
  </main>
</div>
`;

exports[`Index styled component should match snapshot 1`] = `
<div
  className=""
/>
`;
